src/bundle/Resources/public/css/alloyeditor/alloyeditor-ez.css

```diff
@@ -929,7 +929,7 @@
   background-color: #fff;
   border-radius: 3px;
   color: #646464;
-  box-shadow: 0 4px 12px 0 rgba(86, 94, 99, 0.45);
+  box-shadow: 0 4px 10px 0 rgba(86, 94, 99, 0.45);
```

Comment:
@dew326 Could you please add `box-shadow: 0 4px 10px 2px rgba(86, 94, 99, 0.45);`
Thanks!